ambry-tools/src/main/java/com.github.ambry/tools/admin/ConcurrencyTestTool.java

```diff
@@ -236,9 +245,10 @@ public void onCompletion(Pair<String, byte[]> result, Exception exception) {
                 countDownLatch.countDown();
               }
             };
-            putHelper.putBlob(callback);
+            putHelper.putBlob(callback, metricsCollector);
           }
           countDownLatch.await(5, TimeUnit.MINUTES);
+          metricsCollector.updateTimePassedSoFar(SystemTime.getInstance().nanoseconds() - startTimeInMs);
```

Comment:
Actually never mind - I am making changes to this file as part of an open PR, will put in the fix.